lib/options/dcu.robot

```diff
@@ -19,6 +19,7 @@ Set UEFI Option
     Switch To Root User
     Get Flashrom From Cloud
     Execute Command In Terminal    flashrom -p internal -r coreboot.rom --fmap -i FMAP -i SMMSTORE &> /dev/null
+    Execute Command In Terminal    chmod 666 coreboot.rom
```

Comment:
:smiling_imp:

Reply:
But why it was needed?

Reply:
@macpijan whenever 
* `coreboot.rom` doesn't exist yet
* you run sudo su && read with flashrom to `coreboot.rom`

`coreboot.rom` gets created with root only access, and there's `Permission Denied` trouble later on with modifying and sending it